### ISSUE TITLE
v252 release tarball contains problematic hardlink

### ISSUE THREAD

**Issue by mtorromeo** [cockpit-project/cockpit-machines#379]
Whlie extracting cockpit-machines-252.tar.xz I get the following error:

```
cockpit-machines/src/lib/cockpit-po-plugin.js: Skipping hardlink pointing to itself: cockpit-machines/src/lib/cockpit-po-plugin.js
bsdtar: Error exit delayed from previous errors.
```

**Comment by mtorromeo** [comment]
Same deal with cockpit-podman-35.tar.xz, I'm not sure it's worth a separate issue:

```
cockpit-podman/src/lib/cockpit-po-plugin.js: Skipping hardlink pointing to itself: cockpit-podman/src/lib/cockpit-po-plugin.js
bsdtar: Error exit delayed from previous errors.
```

**Comment by martinpitt** [comment]
Confirmed:
```
❱❱❱ make dist; tar tvf cockpit-machines-252.4.g1cf89e4a.tar.xz|grep po-plugin
-rw-r--r-- martin/martin  5261 2021-09-16 08:44 cockpit-machines/src/lib/cockpit-po-plugin.js
hrw-r--r-- martin/martin     0 2021-09-16 08:44 cockpit-machines/src/lib/cockpit-po-plugin.js link to cockpit-machines/src/lib/cockpit-po-plugin.js
```
which is weird.. src/lib/cockpit-po-plugin.js is a regular file, both in cockpit itself and the partial checkout that c-{machines,podman} do. I'll take a closer look, thanks for the report!

**Comment by martinpitt** [comment]
Fixed in https://github.com/cockpit-project/starter-kit/pull/491 . Once that lands, I'll backport the commit to -machines and -podman.

**Comment by martinpitt** [comment]
Picked up in podman in https://github.com/cockpit-project/cockpit-podman/pull/765
